examples/pylink-rtt

```diff
@@ -148,10 +148,12 @@ def main(target_device, block_address=None):
         while jlink.connected():
             time.sleep(1)
         print("JLink disconnected, exiting...")
+    except EOFError:
```

Comment:
Does Windows treat `KeyboardInterrupt` as `EOFError`? I.e. `Ctrl-C` leads to `EOFError` on Windows? If so, I would just ignore printing that `EOFError` was detected, as we just care about `Ctrl-C`.

Reply:
Yes. `EOFError` occured when `builtins.input()` detect `KeyboardInterrupt`.
refer: https://docs.python.org/3/library/functions.html#input

And never return like bellow if not catch `EOFError`.
The code wasn't good placed so I move it closer to the input.

```
IO write thread exception, exiting...
Exception ignored in thread started by: <function write_rtt at 0x0000022C4272A550>
ctrl-c detected, exiting...
Traceback (most recent call last):
  File "XXX\pylink\examples\pylink-rtt", line 87, in write_rtt
Traceback (most recent call last):
  File "XXX\pylink\examples\pylink-rtt", line 149, in main
    bytes = list(bytearray(input(), "utf-8") + b"\x0A\x00")
EOFError:
    time.sleep(1)
KeyboardInterrupt

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "XXX\pylink\examples\pylink-rtt", line 167, in <module>
    sys.exit(main(args.target_cpu, args.rtt_block_address))
  File "XXX\pylink\examples\pylink-rtt", line 152, in main
    print("ctrl-c detected, exiting...")
KeyboardInterrupt
```